### ISSUE TITLE
Incomplete documentation regarding zstd filter

### ISSUE THREAD

**Issue by KBentley57** [boostorg/iostreams#96]
The zst filter does not have a documentation page similar to bzip2, gzip, or zlib at 
https://www.boost.org/doc/libs/1_70_0/libs/iostreams/doc/classes.

**Comment by marxin** [comment]
I can confirm it's a pity that the project does not document the nice feature.
@rdoeffinger: Is it something you would be interested in?

**Comment by rdoeffinger** [comment]
No, I don't have the time or experience to write and maintain documentation hand-written in HTML. It's not a documentation format I've ever used, or would ever want to use to be honest.

**Comment by marxin** [comment]
Sure, understood! HTML is not the friendlies documentation format.
